sharething_pwa/src/AddItem/AddItemScreen.tsx

```diff
@@ -53,7 +63,7 @@ class AddItemScreen extends React.Component<Props, State> {
                 <Button variant="primary" type="submit">
                     Submit
           </Button>
-                {/* {error && <p>{error.message}</p>} */}
+                {this.state.error && <p>{this.state.error.message}</p>}
```

Comment:
Does this compile :)? You defined `error` as `string`, not an object with `message` property